ex_feels/lib/ex_feels_web/twitter/crypto.ex

```diff
@@ -0,0 +1,85 @@
+defmodule ExFeelsWeb.Twitter.Crypto do
+  # https://developer.twitter.com/en/docs/basics/authentication/guides/authorizing-a-request
+
+  @consumer_key Application.get_env(:ex_feels, :twitter)[:consumer_key]
+  @consumer_secret Application.get_env(:ex_feels, :twitter)[:consumer_secret]
+
+  @access_token Application.get_env(:ex_feels, :twitter)[:access_token]
+  @token_secret Application.get_env(:ex_feels, :twitter)[:token_secret]
+
+  def oauth_header(method, url) do
+    nonce = nonce()
+    ts = timestamp()
+
+    sig =
+      {nonce, ts}
+      |> signature_base_string(method, url)
+      |> generate_signature()
+
+    'OAuth oauth_consumer_key="#{consumer_key()}",oauth_token="#{token()}",oauth_signature_method="#{method()}",oauth_timestamp="#{ts}",oauth_nonce="#{nonce}",oauth_version="#{version()}",oauth_signature="#{sig}"'
```

Comment:
This is the only line exceeding my ruler. I tried to break it a part but that added newlines and broke the request :rage1: